packages/core/app/styles/navi-core/components/navi-visualization-config/line-chart.less

```diff
@@ -4,37 +4,8 @@
  */
 
 .line-chart-config {
-  .config-container();
   padding: 5px;
 
-  &__header {
-    color: @navi-gray;
-    margin: 5px;
-  }
-
-  &__style-options {
-    .config-section();
-  }
-
-  &__curve-opt,
-  &__area-opt,
-  &__stacked-opt {
-    align-items: center;
-    display: flex;
-    justify-content: space-between;
-    margin: 8px 0;
-    &-title {
-      text-align: left;
-    }
-  }
-
-  &__area-opt,
-  &__stacked-opt {
-    &-select {
-      padding-right: 8px;
-    }
-  }
-
```

Comment:
Styles below still needed for `__curve-opt`?

Reply:
Those are for the stacked option, for displaying all of the columns, I'll delete a few and see what happens

Reply:
Those are all necessary for the stack order styling